pkg/deploy/api/v1beta1/defaults.go

```diff
@@ -0,0 +1,49 @@
+package v1beta1
+
+import (
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+
+	deployapi "github.com/openshift/origin/pkg/deploy/api"
+)
+
+func init() {
+	mkintp := func(i int) *int64 {
+		v := int64(i)
+		return &v
+	}
+
+	defaultRollingParams := &deployapi.RollingDeploymentStrategyParams{
```

Comment:
You can't do this - all objects will end up sharing the same copy of deployment strategy params.